### DISCUSSION TITLE
Can we have control center slider support?

### DISCUSSION THREAD

**Discussion by msdurex** [MonitorControl/MonitorControl#1562] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Is your feature request related to a problem? Please describe

Monitor and Control app works prefect on my external display setup. 

### Describe the solution you'd like

Extra icon on menu bar is too crowded.

### Describe alternatives you've considered

 If it's possible, control center slider support will be so nice.

**Comment by waydabber** [comment]
No, sadly the Control Center is not expandable afaik, it's a locked down part of the OS.

**Reply by melyux** [reply]
I think it's expandable now. Since Menu Bar real estate is limited, having Monitor Control be in Control Center would save lives. That notch doesn't let us have a lot of stuff on the Menu Bar anymore

**Reply by waydabber** [reply]
Well, I think you can add buttons in Tahoe. No complex controls like sliders last time I checked. But I might be wrong about that.
